staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/helpers.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"fmt"
+
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+
+	apiextensionsfeatures "k8s.io/apiextensions-apiserver/pkg/features"
+)
+
+// GetCRDSchemaForVersion returns the validation schema for given version in given CRD Spec.
+func GetCRDSchemaForVersion(crdSpec *CustomResourceDefinitionSpec, version string) (*CustomResourceValidation, error) {
```

Comment:
this is not called. Drop it.

Reply:
It is used by integration tests, where we use versioned types https://github.com/kubernetes/kubernetes/pull/67553/commits/b0cc5dff010aeaacc0de424969b90080dd75a0df#diff-a1418db843e532f56ceee297e82cf26eR372

Reply:
I commented somewhere else on this. If it is only used in test, move to to test code please. Our production code should not deal with external versions.